Cargo.lock

```diff
@@ -549,6 +549,28 @@ dependencies = [
  "os_str_bytes",
 ]
 
+[[package]]
+name = "clipboard"
+version = "0.5.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "25a904646c0340239dcf7c51677b33928bf24fdf424b79a57909c0109075b2e7"
+dependencies = [
+ "clipboard-win",
+ "objc",
+ "objc-foundation",
+ "objc_id",
+ "x11-clipboard",
+]
+
+[[package]]
+name = "clipboard-win"
+version = "2.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e3a093d6fed558e5fe24c3dfc85a68bb68f1c824f440d3ba5aca189e2998786b"
+dependencies = [
+ "winapi 0.3.9",
+]
+
```

Comment:
Why?

Reply:
Hm? I just added a `clipboard` crate for the write the content to OS's Clipboard.

Reply:
We don't need more dependencies, clipboard is already provided via druid
https://github.com/lapce/lapce/blob/d695914acd737083561eac1bb17b916b53214901/lapce-data/src/document.rs#L58

Reply:
> We don't need more dependencies, clipboard is already provided via druid
> 
> https://github.com/lapce/lapce/blob/d695914acd737083561eac1bb17b916b53214901/lapce-data/src/document.rs#L58

Oh, okay. Didn't know. Okay, I will fix this.